docs/ldml/tr35.md

```diff
@@ -2972,6 +3033,40 @@ The binary operators '&', '-', and the implicit union have equal precedence and
 
 **One caution:** the '&' and '-' operators operate between sets. That is, they must be immediately preceded and immediately followed by a set. For example, the pattern **[[:Lu:]-A]** is illegal, since it is interpreted as the set **[:Lu:]** followed by the incomplete range **-A**. To specify the set of upper case letters except for 'A', enclose the 'A' in brackets: **[[:Lu:]-[A]]**.
 
+##### <a name="Variables_in_UnicodeSets" href="#Variables_in_UnicodeSets">Variables in UnicodeSets</a>
+
+Support for variable identifiers (var) is optional. 
```

Comment:
Add: But when they are used, they are defined as follows:

Reply:
Thank you, added!